scripts/seedDb.mjs

```diff
@@ -51,3 +56,25 @@ async function recreateLeagueData(leagueKeyPrefix, dataRepo) {
         await redis.json.set(leagueKeyPrefix+user.userId, "$", userString)
     }
 }
+
+// Create league configuration data
+
+await recreateLeagueConfigurationData("league_configuration:amazing_race:35:", amazingRace35LeagueConfiguration)
+await recreateLeagueConfigurationData("league_configuration:amazing_race:36:", amazingRace36LeagueConfiguration)
+await recreateLeagueConfigurationData("league_configuration:big_brother:26:", bigBrother26LeagueConfiguration)
+await recreateLeagueConfigurationData("league_configuration:survivor:47:", survivor47LeagueConfiguration)
+
+async function recreateLeagueConfigurationData(leagueKeyPrefix, dataRepo) {
+
+    const leagueConfigurationCursor = await redis.scan("0", {match: leagueKeyPrefix+"*"})
+
+    for (const aKey of leagueConfigurationCursor[1]) { //list of keys
+        redis.del(aKey)
+    }
+
+    const leagueConfigString = JSON.stringify(dataRepo)
+
+    console.log(`Setting league configuration data for '${leagueKeyPrefix}'`);
+    
+    await redis.json.set(leagueKeyPrefix, "$", leagueConfigString)
```

Comment:
I would actually say that we don't want the the key to be the prefix here. The prefix is everything before the final value in the key. In fact I think our "prefixes" here probably aren't prefixes at all, they are just keys.

Reply:
Updated and done